Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -52,13 +52,13 @@ import BraintreeCore
 
     /// Initialize a new Venmo client instance.
     /// - Parameters:
-    ///   - apiClient: The API Client
+    ///   - authorization: A client token or tokenization key
```

Comment:
It looks like we are using a few different strings for this. Are we able to align them across the latest from v7?